packages/x-components/src/store/utils/status-store.utils.ts

```diff
@@ -31,7 +31,7 @@ export interface StatusMutations {
  *
  * @public
  */
-export type RequestStatus = 'success' | 'loading' | 'error';
+export type RequestStatus = 'success' | 'loading' | 'error' | '';
```

Comment:
Just to keep the semantic naming, I would probably use `'initial'` instead of `''`